corehq/apps/fixtures/tests/test_download.py

```diff
@@ -180,6 +180,30 @@ def test_get_location_names(self):
         self.assertEqual(names.get_location_codes(rows[2]), ['3'])
         self.assertEqual(len(rows), 3)
 
+    def test_with_missing_owners(self):
```

Comment:
nit: it seems like this tests 2 things at once? That rows that are missing owners in the uploaded lookup table handle that accordingly, and that rows that have owners that have since been deleted handle that properly? I could be misunderstanding that, but if the test could be split up that would be great.

Reply:
You are correct, and I considered this, but the setup is the bulk of the test, and it didn't feel worth it to duplicate all of that setup to test counting and actual names separately, so I tacked three more assertions onto the end of the counting test. In this case I think practicality beats purity, especially since the setup is relatively slow because of `self.upload(...)`.